### DISCUSSION TITLE
BUG: Set release dates/legality out of line.

### DISCUSSION THREAD

**Discussion by nebman227** [glacials/whatsinstandard#290] [General]
Per the current MTR, cards from new sets are legal in all formats that they'll be legal in at Prerelease time. Duskmourn has been legal since Friday (and has already been played with in comp events!). The website seems to use release date for legality and says Duskmourn is 3 days out from being legal, which is no longer true.

**Comment by glacials** [comment]
Agreed! #270 agrees with you as well. To cite the specific section of the [Magic Tournament Rules](https://media.wizards.com/ContentResources/WPN/MTG_MTR_2024_Aug26.pdf), section 3.7 has two upcoming sets' specific legality dates which match up with their prerelease dates:

<img width="1259" alt="image" src="https://github.com/user-attachments/assets/e19a6343-90a2-4ce4-bc57-605bac10524a">

I don't see a general statement that prerelease dates are also rotation dates, but this is good enough for me.

Does this change how drops work, or do drops still happen on formal release days, causing Standard to have up to 13 sets in it for a one week period?
